k8s/configuration/config-map.yaml

```diff
@@ -0,0 +1,36 @@
+#######################################################
+# This YAML file desribes a ConfigMap for the cluster #
```

Comment:
The opening comment indicates there is one ConfigMap for the whole cluster.

But then there are four different ConfigMaps in this file. Why not just have one ConfigMap for the entire Kubernetes cluster? (A ConfigMap can contain several key-value pairs.)